### ISSUE TITLE
Use iam.assumeRolePolicyForPrincipal function when merged into pulumi-aws

### ISSUE THREAD

**Issue by jen20** [jen20/pulumi-aws-vpc#1]
Use helper functions rather than hand-constructing role assumption policies, once https://github.com/pulumi/pulumi-aws/pull/273 is merged.

**Comment by jen20** [comment]
Fixed by #7.
